libnetwork/networkdb/networkdb.go

```diff
@@ -348,7 +348,7 @@ func (nDB *NetworkDB) GetEntry(tname, nid, key string) ([]byte, error) {
 }
 
 func (nDB *NetworkDB) getEntry(tname, nid, key string) (*entry, error) {
-	e, ok := nDB.indexes[byTable].Get(fmt.Sprintf("/%s/%s/%s", tname, nid, key))
+	e, ok := nDB.indexes[byTable].Get([]byte(fmt.Sprintf("/%s/%s/%s", tname, nid, key)))
```

Comment:
We should probably also consider using concatenation for these simple cases (instead of `Sprintf()`)

Reply:
I don't think it's worth to sacrifice readability in this case.